tools/public_api_guard/material/autocomplete.d.ts

```diff
@@ -1,31 +1,11 @@
-export declare const AUTOCOMPLETE_OPTION_HEIGHT = 48;
-
-export declare const AUTOCOMPLETE_PANEL_HEIGHT = 256;
-
-export declare function getMatAutocompleteMissingPanelError(): Error;
-
-export declare const MAT_AUTOCOMPLETE_DEFAULT_OPTIONS: InjectionToken<MatAutocompleteDefaultOptions>;
-
-export declare function MAT_AUTOCOMPLETE_DEFAULT_OPTIONS_FACTORY(): MatAutocompleteDefaultOptions;
-
-export declare const MAT_AUTOCOMPLETE_SCROLL_STRATEGY: InjectionToken<() => ScrollStrategy>;
-
-export declare function MAT_AUTOCOMPLETE_SCROLL_STRATEGY_FACTORY(overlay: Overlay): () => ScrollStrategy;
-
-export declare const MAT_AUTOCOMPLETE_SCROLL_STRATEGY_FACTORY_PROVIDER: {
-    provide: InjectionToken<() => ScrollStrategy>;
-    deps: (typeof Overlay)[];
-    useFactory: typeof MAT_AUTOCOMPLETE_SCROLL_STRATEGY_FACTORY;
-};
-
-export declare const MAT_AUTOCOMPLETE_VALUE_ACCESSOR: any;
-
-export declare class MatAutocomplete extends _MatAutocompleteMixinBase implements AfterContentInit, CanDisableRipple, OnDestroy {
+export declare abstract class _MatAutocompleteBase extends _MatAutocompleteMixinBase implements AfterContentInit, CanDisableRipple, OnDestroy {
```

Comment:
The API differences here come from the fact that I moved all the logic into a base class, but everything should be backwards-compatible.